clusterloader2/GETTING_STARTED.md

```diff
@@ -0,0 +1,239 @@
+```
+
+At the end of clusterloader output you should see pod startup latency and information that test succeeded:
+```
+PodStartupLatency: perc50: 709.983961ms, perc90: 709.983961ms, perc99: 709.983961ms
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s; threshold 2m0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 2s, perc90: 2s, perc99: 2s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s; threshold 2m0s
+--------------------------------------------------------------------------------
+Test Finished
+Test: ./config.yaml
+Status: Success
+--------------------------------------------------------------------------------
+```
+
+If you want to see when each step of your test is executed, you can add `--v=2` argument.
+
+## Delete kind cluster
+To delete kind cluster, run:
+```bash
+kind delete cluster --name test-cluster
+```
+
+## Running 100-node scale test on GKE
+
+Here you can find general purpose [Load test]. It can be used to test clusters starting from 100 nodes up to 5k nodes.
+
+First, we need to create 100-node cluster.
+```bash
+export ZONE=us-central1-c
+export PROJECT=ENTER_YOUR_PROJECT
+export NETWORK_NAME=cluster-network
+export CLUSTER_NAME=scale-cluster
+gcloud config set project $PROJECT
+gcloud compute networks create $NETWORK_NAME --project=$PROJECT --subnet-mode=custom
+gcloud beta container clusters create $CLUSTER_NAME \
+    --quiet \
+    --enable-ip-alias \
+    --create-subnetwork range=/20 \
+    --cluster-ipv4-cidr=/11 \
+    --services-ipv4-cidr=/18 \
+    --enable-private-nodes \
+    --no-enable-master-authorized-networks \
+    --master-ipv4-cidr 172.16.0.0/28 \
+    --enable-autorepair \
+    --project=$PROJECT \
+    --zone=$ZONE \
+    --machine-type=e2-medium \
+    --image-type=gci \
+    --num-nodes=100 \
+    --network=$NETWORK_NAME \
+    --disk-size=50GB
+```
+
+You can run cluster scale test with this command:
+```bash
+KUBE_GKE_NETWORK=$NETWORK_NAME CL2_LATENCY_POD_CPU=80 CL2_SCHEDULER_THROUGHPUT_THRESHOLD=0 ./run-e2e-with-prometheus-fw-rule.sh cluster-loader2 --testconfig=./testing/load/config.yaml --nodes=100 --provider=gke --testoverrides=./testing/experiments/use_simple_latency_query.yaml --testoverrides=./testing/prometheus/not-scrape-kube-proxy.yaml --enable-prometheus-server=true --kubeconfig=${HOME}/.kube/config --v=2
+```
+
+Alternatively, if you want to run clusterloader2 directly, you will need to create firewall rule for proetheus manually:
```

Comment:
How do I even know which one I should choose?

Let's be opinionated and show a single option that we believe is better.